### ISSUE TITLE
placing JSON schema cache dir in $HOME does not start LSP

### ISSUE THREAD

**Issue by chr0n1x** [anasinnyk/nvim-k8s-crd#1]
my config:

```lua
  -- in my packer.nvim config
  use {
    'anasinnyk/nvim-k8s-crd',
    requires = { 'neovim/nvim-lspconfig' },
    config = function()
      require('k8s-crd').setup({
        cache_dir = vim.fn.expand("$HOME/.cache/nvim-k8s-crd/"),
        k8s = { file_mask = "*.yaml", },
      })
    end
  }
```

what ends up happening is that `$HOME/.cache/nvim-k8s-crd/` ends up being expanded, but then the LSP server never pulls any data while editing any files. however if I change the dir to `.cache/nvim-k8s-crd/`, all JSON schemas get downloaded to my `cwd`, and the LSP loads in suggestions/annotations properly.

very weird.

_PS: this plugin is actually really friggen awesome, great work and thank you!_

**Comment by anasinnyk** [comment]
Hi, thanks for you request.
Please try use `~` sign instead `$HOME`, HOME here is environment variable, and it doesn't exist in lua runtime.

**Comment by anasinnyk** [comment]
I think this issue was fixed i #3 pull request, pls check, and reopen if I'm wrong
